src/user/models/user-settings.entity.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { UserEntity } from './user.entity';
+
+@Entity('user_settings')
+export class UserSettingsEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @OneToOne(() => UserEntity)
+  @JoinColumn()
+  user_: UserEntity;
+
+  @Column({ default: 'default' })
+  theme: string;
```

Comment:
TODO later: let's ENUM that one 😄

Reply:
This and some others (like Blood Type for donations, Media Type for social posts, etc.) will indeed be enums